test-network-nano-bash/network.sh

```diff
@@ -82,19 +87,26 @@ networkStart() {
   sleep ${CLI_DELAY}
 
   if [ "${CREATE_CHANNEL}" = "true" ]; then
-    echo "Creating channel (peer1)..."
-    . ./peer1admin.sh && ./create_channel.sh
-
-    echo "Joining channel (peer2)..."
-    . ./peer2admin.sh && ./join_channel.sh
-
-    echo "Joining channel (peer3)..."
-    . ./peer3admin.sh && ./join_channel.sh
-
-    echo "Joining channel (peer4)..."
-    . ./peer4admin.sh && ./join_channel.sh
+    if [ "$ORDERER_TYPE" = "smartbft" ]; then
```

Comment:
can we simplify and always use osnadmin and channel participation also for Raft?